examples/basic-jsx-external/index.html

```diff
@@ -10,12 +10,16 @@
     <h1>Basic Example with External JSX</h1>
     <div id="container">
       <p>
-        If you can see this, React is not working right. This is probably because you&apos;re viewing
-        this on your file system instead of a web server. Try running
+        If you can see this, React is not working right. This is probably because you&apos;re viewing this on your file system instead of a web server.<br>
+        Try navigating to the React root 
+        <pre>
+          cd /path/to/react
+        </pre>
+        and running
         <pre>
           python -m SimpleHTTPServer
         </pre>
-        and going to <a href="http://localhost:8000/">http://localhost:8000/</a>.
+        and going to <a href="http://localhost:8000/examples/basic-jsx-external/">http://localhost:8000/examples/basic-jsx-external/</a>.
```

Comment:
@zpao I tend to agree with @MattijsKneppers here - I'd rather that the examples be clear.  I also ran into this issue, but looking at the source I quickly figured out what was wrong and ran the webserver from the correct directory - but it might not be as obvious to some new users.  Having some better usage instructions in the html seems like a harmless enhancement.

If you want to have super strong consistency, we could add the instructions to all the files, though that seems wholly unnecessary since many of the files don't actually require a web server.